<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QTextBrowser" name="msg_txt_browser_">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>531</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="msg_txt_edit_">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>531</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="usr_tbl_widget">
   <property name="geometry">
    <rect>
     <x>555</x>
     <y>10</y>
     <width>161</width>
     <height>401</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="showGrid">
    <bool>false</bool>
   </property>
   <column>
    <property name="text">
     <string>User Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>IP Address</string>
    </property>
   </column>
  </widget>
  <widget class="QFontComboBox" name="font_cbx_">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>271</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="size_cbx_">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>271</y>
     <width>69</width>
     <height>31</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>4</number>
   </property>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>11</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>12</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>13</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>14</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>15</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>16</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>17</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>18</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>19</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>20</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>21</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>22</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="send_btn_">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>420</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QLabel" name="usr_num_lbl_">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>420</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="exit_btn_">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>420</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bold_btn_">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>270</y>
     <width>33</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>bold</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/text-bold-icon.png</normaloff>:/images/text-bold-icon.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>26</width>
     <height>26</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="italic_btn_">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>270</y>
     <width>33</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>italic</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/text-italic-icon.png</normaloff>:/images/text-italic-icon.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>26</width>
     <height>26</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="under_btn_">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>270</y>
     <width>33</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>under</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/text-underline-icon.png</normaloff>:/images/text-underline-icon.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>26</width>
     <height>26</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="color_btn_">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>270</y>
     <width>33</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>font color</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/Colors-icon.png</normaloff>:/images/Colors-icon.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>26</width>
     <height>26</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="send_file_btn_">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>270</y>
     <width>33</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>send file</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/Send-Document-icon.png</normaloff>:/images/Send-Document-icon.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>26</width>
     <height>26</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="save_file_btn_">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>270</y>
     <width>33</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>save chat logs</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/save-diskette-floppy-disk-icon.png</normaloff>:/images/save-diskette-floppy-disk-icon.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>26</width>
     <height>26</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="clear_btn_">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>270</y>
     <width>33</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>empty chat logs</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/broom-icon.png</normaloff>:/images/broom-icon.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>26</width>
     <height>26</height>
    </size>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
